test/integration/targets/azure_rm_virtualmachine/tasks/virtualmachine.yml

```diff
@@ -251,6 +256,82 @@
       state: absent
       vm_size: Standard_A0
 
+- name: Create NICs for dual nic VM
+  azure_rm_networkinterface:
+      resource_group: "{{ resource_group }}"
+      name: "{{ item }}"
+      virtual_network: testvm001
+      subnet: testvm001
+      security_group: testvm001
+  loop:
+    - testvm011
+    - testvm012
+
+- name: Create virtual machine with two NICs
```

Comment:
it will be more nice to have a test covering create vm with one nic, then update to multiple nic